pos_quick_logout/static/src/xml/pos_quick_logout.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!--
     Copyright (C) 2016-Today: La Louve (<http://www.lalouve.net/>)
+    Copyright (C) 2019-Today: Druidoo (<https://www.druidoo.io>)
```

Comment:
Not an expert, legally, but I don't think that include copyright if you did'nt touched the file is valid.
Generaly, an OCA convention is to include copyright if you touched "significantly" the code. (not for file or argument rename, ORM changes, new OCA rules)